src/components/Header.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { useEffect } from "react";
+import { FaShoppingCart, FaHome } from "react-icons/fa";
+import { NavLink, useNavigate, useLocation} from "react-router-dom";
+
+
+const Header = ({ page, setPage }) => {
+    const navigate = useNavigate();
+    const location = useLocation();
+
+    useEffect(() => {
+        if (location.pathname === '/') navigate("/products/electronics");
+    }, [navigate, location.pathname]);
```

Comment:
functionality wise this is okay , but this is not good ux , user will see loading two time due to redirect